app/javascript/lib/cell-faceting.js

```diff
@@ -276,7 +294,7 @@ function getCellsByFacet(filterableCells, annotationFacets) {
   const cellsByFacet = {}
   for (let i = 0; i < annotationFacets.length; i++) {
     const facet = annotationFacets[i]
-    const facetCrossfilter = cellCrossfilter.dimension(d => d[facet])
+    const facetCrossfilter = cellCrossfilter.dimension(d => d.facetIndex[i])
```

Comment:
FWIW this is still the line where I get the "pre-crash" pause in the DevTools debugger when looking at very large studies in the cell filtering UX.

Reply:
Thanks, this is useful to know.  The containing function here is still probably being called too frequently.  So although this function is a lot lighter, it's still not light enough to deal with many quick filter clicks in our biggest 1% of studies.